File: ABC200/c.py
# -*- coding: utf-8 -*-
import numpy as np
n = int(input())

tmp = list(map(int, input().split()))
tmp_np = np.array(tmp)

ans = 0

for i in range(1, int(n/2)):
    
    ans =ans + sum((tmp_np[i-1]-tmp_np[i:])%200==0)*2
    
    if any((tmp_np[i-1]-tmp_np[i:])%200==0) and i<n/2:
        ans =ans -1
        print(i)
    
    print((tmp_np[i-1]-tmp_np)%200==0)

print(ans)

File: ABC200/c.cc
#include <iostream>

int main(){
  long long int n, array[2*100000], i, j;
  int ans = 0;
  std::cin >> n;

  for(i=0; i<n; i++){
    std::cin >> array[i];
  }

  for(i=0; i<(int)n/2; i++){
    for(j=0; j<n; ++j){
      if((array[i]-array[j])%200 == 0){
        ans +=2;
        if(j<=(int)((n-1)/2)){
          ans -=1;
        }

      }
    }
  }
std::cout << ans << '\n';
  return 0;
}
